### ISSUE TITLE
Django 3.2 issue

### ISSUE THREAD

**Issue by lampslave** [Kub-AT/django-cache-fallback#3]
Please use django.VERSION to check version in cache_fallback.cache.get_cache because django.core.cache doesn't have `_create_cache` anymore.

**Comment by cjw296** [comment]
Also seeing:
```
Traceback (most recent call last):
  File "...manage.py", line 11, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/lib/python3.6/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/usr/lib/python3.6/site-packages/django/core/management/__init__.py", line 413, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/lib/python3.6/site-packages/django/core/management/base.py", line 354, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/lib/python3.6/site-packages/django/core/management/base.py", line 398, in execute
[2021-06-23T05:09:06][DEBUG] urllib3.connectionpool connectionpool.py:_new_conn:975 | Starting new HTTPS connection (1): sentry.w2k.jumptrading.com:443
    output = self.handle(*args, **options)
  File "/usr/lib/python3.6/site-packages/django/core/management/base.py", line 89, in wrapped
    res = handle_func(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/django/core/management/commands/migrate.py", line 75, in handle
    self.check(databases=[database])
  File "/usr/lib/python3.6/site-packages/django/core/management/base.py", line 423, in check
    databases=databases,
  File "/usr/lib/python3.6/site-packages/django/core/checks/registry.py", line 76, in run_checks
    new_errors = check(app_configs=app_configs, databases=databases)
  File "/usr/lib/python3.6/site-packages/django/core/checks/caches.py", line 63, in check_file_based_cache_is_absolute
    cache = caches[alias]
  File "/usr/lib/python3.6/site-packages/django/utils/connection.py", line 62, in __getitem__
    conn = self.create_connection(alias)
  File "/usr/lib/python3.6/site-packages/django/core/cache/__init__.py", line 44, in create_connection
    return backend_cls(location, params)
  File "/usr/lib/python3.6/site-packages/cache_fallback/cache.py", line 40, in __init__
    self._cache = get_cache('main_cache')
  File "/usr/lib/python3.6/site-packages/cache_fallback/cache.py", line 23, in get_cache
  ```

**Comment by cjw296** [comment]
This is the django commit that removed it: https://github.com/django/django/commit/148702e7258eafb27d5488b33a723c87d898e22b#diff-08c0d636d56b6cc14f8268c913713817487830fa6c3ac60e1aeba9526e9b5793

**Comment by Kub-AT** [comment]
@lampslave @cjw296

check it now, after this fix: https://github.com/Kub-AT/django-cache-fallback/commit/db2e88b3ac2c0361fc7d0b4bae0af030809abfce#diff-5e0098f1bb2518012991963db9dde9b04eb9a79e347891dc3905774c37190979

**Comment by imhmdb** [comment]
> @lampslave @cjw296
> 
> check it now, after this fix: [db2e88b#diff-5e0098f1bb2518012991963db9dde9b04eb9a79e347891dc3905774c37190979](https://github.com/Kub-AT/django-cache-fallback/commit/db2e88b3ac2c0361fc7d0b4bae0af030809abfce#diff-5e0098f1bb2518012991963db9dde9b04eb9a79e347891dc3905774c37190979)

can we have a new release with this change?

**Comment by Kub-AT** [comment]
@imhmdb 0.3.0 published
https://pypi.org/project/django-cache-fallback/0.3.0/
